src/wallet/spend.cpp

```diff
@@ -174,6 +174,11 @@ TxSize CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *walle
     CTransaction ctx(txNew);
     int64_t vsize = GetVirtualTransactionSize(ctx);
     int64_t weight = GetTransactionWeight(ctx);
+    // ELEMENTS: use discounted vsize for CTs if enabled
+    if (Params().GetCreateDiscountCT()) {
```

Comment:
I think it doesn't make any sense to CreateDiscountCT while not AcceptDiscountCT.

I think you will need to add GetAcceptDiscountCT to this `if` statement or otherwise make the configuration of CreateDiscountCT without AcceptDiscountCT illegal.

Reply:
Thanks, agreed. I think it's best that if "create" is true then "accept" is also set to true. 

Done in 7fc9651